src/polkitagent/polkitagenthelper-pam.c

```diff
@@ -113,7 +127,47 @@ main (int argc, char *argv[])
       goto error;
     }
 
```

Comment:
Hi Luca, formally it all looks good, thanks for the hard work! Just to clear out the purpose, we still have a SUID binary (because of pam stack), but at least it runs in systemd-sandbox, right?

Reply:
Socket activation is manual on purpose, or `WantedBy` is missing?

Reply:
It's manual on purpose on the service, as that is only triggered by the socket unit

Reply:
Yes the socket unit itself has a WantedBy

Reply:
True, I had it right before my eyes. On the first run on VM, something weird happened and the polkit helper service unit failed after boot for some reason, so I started to wonder... It went all good on second provision.